examples/app/weather/main.py

```diff
@@ -1,3 +1,12 @@
+''' A weather chart for three cities using a csv file.
+This illustration demonstrates different interpretation of the same data
+with the distribution option.
+
+.. note::
+    This chart needs the Scipy and Pandas package to run. See
+    the included `README`_ for more information.
```

Comment:
```suggestion
    ``README.md`` for more information.
```

There's no link alias for README so the trailing underscore has no effect. Suggest using double backticks to render a path code/pre font.